.github/actions/is-release-in-progress-v1/src/is-release-in-progress.ts

```diff
@@ -0,0 +1,23 @@
+import type { PullRequest } from './types'
+
+/**
+ * Check if there is a release pull request
+ * by checking the labels of pull requests given.
+ * 
+ * For reference of pull request schema, see response schema
+ * of list pull requests endpoint of GitHub REST API
+ 
+ * @see https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#list-pull-requests
+ *
+ * @param pullRequests List of pull requests
+ * @returns TRUE if there is a release pull request
+ */
+export default function isReleaseInProgress(
+  pullRequests: PullRequest[]
+): boolean {
+  const releasePullRequests = pullRequests.filter(pr =>
+    pr.labels.some(label => label.name === 'release')
+  )
+
+  return releasePullRequests.length > 0
+}
```

Comment:
We probably don't want to continue looping through the pull requests once we have found one that has the label `release`, we could do something like: 

```suggestion
let hasReleaseLabel = false;

export function isReleaseInProgress(pullRequests: PullRequest[]): boolean {
  for (const pullRequest of pullRequests) {
    // perf: exit early if we already found a release label
    if (hasReleaseLabel) {
      return true;
    }

    hasReleaseLabel = pullRequest.labels.some(
      (label) => label.name.toLowerCase() === "release"
    );
  }

  return hasReleaseLabel;
}
```